src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
+                .chi-form__item
+                  select(class="chi-select", id="example-ba1")
+                    option(value="") Select
+                    option Option 1
+                    option Option 2
+                    option Option 3
+                .chi-form__item
+                  select(class="chi-select", id="example-ba2")
+                    option(value="") Select
+                    option Option 1
+                    option Option 2
+                    option Option 3
+                button.chi-button.-icon.-flat(id="button-b-filters" aria-label="Filters" data-tooltip="Filters" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-filter
+              .chi-toolbar__filters-mobile
+                button#drawer-trigger-filter1.chi-button.-icon.-flat.chi-drawer__trigger(data-target='#drawer-1', aria-label="Button action")
+                  .chi-button__content
+                    i.chi-icon.icon-filter
+          .chi-toolbar__end
+            .chi-toolbar__actions
+              .chi-toolbar__actions-desktop
+                button.chi-button.-icon.-flat(id="button-b-download" aria-label="Download" data-tooltip="Download" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-arrow-to-bottom
+                button.chi-button.-icon.-flat(id="button-b-refresh" aria-label="Refresh" data-tooltip="Refresh" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-refresh
+                button.chi-button.-icon.-flat(id="button-b-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-table-column-settings
+              .chi-toolbar__actions-mobile
+                button#drawer-trigger-actions1.chi-button.-icon.-flat.chi-drawer__trigger(data-target='#drawer-2', aria-label="Button action")
+                  .chi-button__content
+                    i.chi-icon.icon-more-vert
+    .example-tabs.-pl--2
+      ul.chi-tabs#example-base(role="tablist")
+        li.-disabled
+          a(
+            href=`#webcomponent-base`
+            role="tab"
+            aria-selected="false"
+            tabindex="-1"
+            aria-controls="webcomponent-base"
+          ) Web Component
+        li.-active
+          a(
+            href='#vue-interaction'
+            role="tab"
+            aria-selected="true"
+            aria-controls="vue-interaction"
+          ) Vue
+        li
+          a(
+            href=`#html-base`
+            role="tab"
+            aria-selected="false"
+            aria-controls="html-base"
+          ) HTML Blueprint
+    .chi-tabs-panel#webcomponent-base
+    .chi-tabs-panel.-active#vue-interaction
+      .chi-tab__description
+        p.-text
+          | To render toolbar, use the class <code>chi-toolbar</code>.
```

Comment:
The line is `HTML blueprint` specific. Should not be visible in Vue

Reply:
Solved.